test/lib/createWorkers.test.js

```diff
@@ -326,7 +271,7 @@ describe('Worker library', () => {
       expect(message).to.be.false();
     });
 
-    it('should call provided handlers and ack if handlers runs ok', function* test() {
+    it('should call all provided handlers and ack if handlers runs ok', function* test() {
```

Comment:
We are testing the underlying lib here. How about just calling `createWorkers` and checking that we passed the logger and the returned object?

Reply:
well, since the test are here I will leave them here, might be useful for non-reg...